wemake_python_styleguide/visitors/ast/classes.py

```diff
@@ -118,28 +119,21 @@ def _is_correct_base_class(self, base_class: ast.AST) -> bool:
         return False
 
     def _check_getters_setters_methods(self, node: ast.ClassDef) -> None:
-        class_attributes, instance_attributes = classes.get_attributes(
-            node,
-            include_annotated=True,
-        )
-        flat_class_attributes = name_nodes.flat_variable_names(class_attributes)
-
-        attributes_stripped = {
-            class_attribute.lstrip(constants.UNUSED_PLACEHOLDER)
-            for class_attribute in flat_class_attributes
-        }.union({
-            instance.attr.lstrip(constants.UNUSED_PLACEHOLDER)
-            for instance in instance_attributes
-        })
-
-        for method in classes.find_getters_and_setters(node):
-            if method.name[classes.GETTER_LENGTH:] in attributes_stripped:
-                self.add_violation(
-                    oop.UnpythonicGetterSetterViolation(
-                        method,
-                        text=method.name,
-                    ),
-                )
+        getters_and_setters = set(filter(
+            lambda getter_setter: functions.is_method(
```

Comment:
Sorry, I still don't understand this part.

What I see here is basically a `filter` over `getters_setters.find_paired_getters_and_setters(node)` that finds all methods (which I guess is true for 100% of them, am I right?).

Reply:
As I understand from description of task, we need to find paired methods of classes for more strict checking. We can have nested getters and setters and this situation should pass checking.

Reply:
Ok, I see! Thanks a lot!

Then we need a test for this, if it is missing right now.

Reply:
Oh, I see, we have one: https://github.com/wemake-services/wemake-python-styleguide/blob/b27f20a6703f771279d266afe281d054cf187d2f/tests/test_visitors/test_ast/test_classes/test_getter_setter.py#L107